client/lib/plans/index.js

```diff
@@ -204,3 +215,33 @@ export function plansLink( url, site, intervalType ) {
 
 	return url;
 }
+
+export function applyTestFiltersToPlansList( planName ) {
+	const filteredPlanConstantObj = Object.assign( {}, plansList[ planName ] );
+	let filteredPlanFeaturesConstantObj = getPlanFeaturesConstantObj( plansList[ planName ].getFeatures() );
+
+	if ( ! isWordadsInstantActivationEnabled() ) {
+		filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
+			( _, key ) => key !== WORDADS_INSTANT
+		);
+	} else if ( planName === PLAN_PREMIUM ) {
+		filteredPlanConstantObj.getDescription = plansList[ planName ].getDescriptionWithWordAdsInstantActivation;
+	}
+
+	if ( ! isGoogleVouchersEnabled() ) {
```

Comment:
Since `getFeatures` in `plans/constants` is a method, I think it may be clearer to filter the feature lists at that point rather than in this method.